src/kOS.Safe/Execution/CPU.cs

```diff
@@ -15,15 +15,18 @@ namespace kOS.Safe.Execution
 {
     public class CPU : ICpu
     {
-        private enum Status
+        private enum Section
         {
-            Running = 1,
-            Waiting = 2
+            Main = 1,
+            Trigger = 2
         }
```

Comment:
I realized that ``Cpu.Status`` was only ever used in one and one place only and that was to handle the waits the old way.  The new way made it moot.

The new enum, "Section" is just to remember which code section we are in, trigger or main - I tried using "context" but that word is overused in kOS for many different things and it would have been yet another meaning for it.